build-art.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const sharp = require('sharp');
+
+console.log('Sharp Versions:');
+console.log(sharp.versions);
+console.log(`Sharp Concurrency: ${sharp.concurrency()}`);
+
```

Comment:
```suggestion
```

Is there a reason to keep these `console.log()`s? (If there is, never mind then :slightly_smiling_face:)

Reply:
I felt like they are useful to have in the logs